lib/reconfigure.js

```diff
@@ -58,6 +58,19 @@ module.exports = function(config) {
 
       // reconstruct the URL
       config.url = url.format(parsed);
+    } else {
+      var options = getOptions(config);
```

Comment:
This redefinition of variables is causing linting failures.
Probably easiest to hoist them up (as that is what happens under the covers anyway).